src/components/Search/Search.tsx

```diff
@@ -53,30 +51,39 @@ export interface SearchProps extends React.HTMLAttributes<HTMLDivElement> {
 }
 
 const Search = ({
-  useIcon = true,
   onSearch,
   searchResults = [],
   state = 'idle',
   defaultValue = '',
   className,
-  placeholder = 'Type to search',
+  placeholder,
   searchButtonText = 'Search',
-  loadingText = 'Loading...',
+  loadingText = 'Search In Progress...',
   invalidText = 'Invalid search',
   getAllResultsText = (searchValue) => `View all results for ${searchValue}`,
   getAllResultsLink = (searchValue) => `/Search?Search=${searchValue}`,
```

Comment:
I'm seeing some strange changes to the searchValue when we search on it in dev.phillips.com.  I searched for `test test test` and the URL was: https://www.dev.phillips.com/Search?Search=test+test+test+test

Is there something we need to do to the `searchValue` with this default `getAllResultsLink` function?

Reply:
I think they are doing a weird text replace that I would not like to repeat with our search. Right now I just encode the special charaters normally. When the actual search for the maker hits the backend I believe all special characters are removed anyway so I don't think we have a problem here

Reply:
If you replace test+test+test+test with test20%test20%test20%test it returns with the same result set so I think it should be fine to use the encoded value instead of doing our own string replace